man/docker-exec.1.md

```diff
@@ -28,6 +29,9 @@ container is unpaused, and then run
 **-d**, **--detach**=*true*|*false*
    Detached mode: run command in the background. The default is *false*.
 
+**--detach-keys**=[]
+   Define the key sequence to use to detach from the container.
```

Comment:
I'd add 'from' in there. It's the client doing the detaching, not the container, as I understand.

"Define the key sequence which detaches from the container"?